lib/grammers-client/src/types/attributes.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 - developers of the `grammers` project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+use grammers_tl_types as tl;
+
+pub enum Attribute {
+    Audio {
+        duration: i32,
```

Comment:
Would it make sense to provide a `Duration` here, even if we convert it to seconds later?

Reply:
`Duration` accepts u64 for seconds, while the API allows up to i32, which should not be exceeded in a real life scenario, but the type cast is still prone to errors 🤔

Reply:
I'm okay with a panic here. No audio should last that long.

Reply:
Okay, it will panic now at values greater than i32 :: MAX